templates/Copy DICOM Metadata Changes to ADLS Gen2 in Delta Format/manifest.json

```diff
@@ -0,0 +1,44 @@
+    "icons": [
```

Comment:
I manually added the `"WebActivity"` and `"AzureBlobFSFile"` icons to better illustrate what is going on inside of the single `"Until"` activity